server/manifest.go

```diff
@@ -2,10 +2,46 @@
 
 package main
 
-var manifest = struct {
-	ID      string
-	Version string
-}{
-	ID:      "com.mattermost.plugin-channel-export",
-	Version: "1.0.0",
+import (
+	"encoding/json"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+)
+
+var manifest *model.Manifest
+
+const manifestStr = `
+{
+  "id": "com.mattermost.plugin-channel-export",
+  "name": "Channel Export",
+  "description": "This plugin allows channel export into a human readable format.",
+  "homepage_url": "https://github.com/mattermost/mattermost-plugin-channel-export/",
+  "support_url": "https://github.com/mattermost/mattermost-plugin-channel-export/issues",
+  "release_notes_url": "https://github.com/mattermost/mattermost-plugin-channel-export/releases/tag/v1.0.0",
+  "version": "1.0.0",
+  "min_server_version": "5.37.0",
+  "server": {
+    "executables": {
+      "darwin-amd64": "server/dist/plugin-darwin-amd64",
+      "darwin-arm64": "server/dist/plugin-darwin-arm64",
+      "linux-amd64": "server/dist/plugin-linux-amd64",
+      "linux-arm64": "server/dist/plugin-linux-arm64",
+      "windows-amd64": "server/dist/plugin-windows-amd64.exe"
+    },
+    "executable": ""
+  },
+  "webapp": {
+    "bundle_path": "webapp/dist/main.js"
+  },
+  "settings_schema": {
+    "header": "",
+    "footer": "",
+    "settings": []
+  }
+}
+`
+
+func init() {
+	var manifest model.Manifest
```

Comment:
This line need to be removed. Otherwise `manifest` is present in the local scope.

Reply:
thanks for the catch!